app/models/user_presenter.rb

```diff
@@ -54,12 +48,13 @@ def organizations
     return orgs || []
   end
 
-  def my_events
-    Event.my_events(user)
-  end
+  def scrape_stats
+    doc = Nokogiri::HTML(open("https://github.com/#{user.screen_name}"))
+    nums = doc.css('span.contrib-number').children.map { |num| num.text }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Pass `&:text` as an argument to `map` instead of a block.